staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -213,3 +224,48 @@ func (f *FieldManager) buildManagerInfo(prefix string, operation metav1.ManagedF
 	}
 	return internal.DecodeManager(&managerInfo)
 }
+
+// StripFields that don't contribute much value to an objects managedFields
+func (f *FieldManager) StripFields(obj runtime.Object, typed typed.TypedValue, apiVersion fieldpath.APIVersion, managed fieldpath.ManagedFields, manager string) error {
+	set, err := typed.ToFieldSet()
+	if err != nil {
+		return fmt.Errorf("failed to get fieldset: %v", err)
+	}
+
+	strippedFields := &unstructured.Unstructured{
```

Comment:
I was expecting this code to use [MakePath](https://github.com/kubernetes-sigs/structured-merge-diff/blob/master/fieldpath/path.go#L45) to build a couple of path into a `fieldpath.Set` directly and return the difference. No conversion, no new typed, no object, no possible errors?